subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -818,7 +896,161 @@ static void isr_rx(void *param)
 			lll->bis_curr = lll->num_bis;
 		}
 	}
+#endif /* CONFIG_BT_CTLR_SYNC_ISO_SEQUENTIAL */
+
+	goto isr_rx_ctrl;
+
+#if defined(CONFIG_BT_CTLR_SYNC_ISO_INTERLEAVED)
+isr_rx_interleaved:
```

Comment:
This looks like a weird flow to me

So if we reach 901, then we go somewhere else, and we go to `isr_rx_interleaved` above. 

Why not just put the code for `isr_rx_interleaved` into the `if` above instead of having the `goto`? 

(Oh, I see that the below recursively go up here too... Very difficult flow to follow; consider if it can be done differently).